config/resources/domain.json

```diff
@@ -806,6 +767,68 @@
       },
       "new-resource-base": "http://data.lblod.info/id/vestigingen/"
     },
+    "memberships": {
+      "class": "org:Membership",
+      "relationships": {
+        "member": {
+          "predicate": "org:member",
+          "target": "organizations",
+          "cardinality": "one"
+        },
+        "organization": {
+          "predicate": "org:organization",
+          "target": "organizations",
+          "cardinality": "one"
+        },
+        "role": {
+          "predicate": "org:role",
+          "target": "membership-roles",
+          "cardinality": "one"
+        },
+        "during": {
+          "predicate": "euro:memberDuring",
+          "target": "period-of-time",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/id/lidmaatschap/"
+    },
+    "membership-roles": {
+      "class": "org:Role",
+      "attributes": {
+        "label": {
+          "predicate": "skos:prefLabel",
+          "type": "string"
+        }
+      },
+      "relationships": {
+        "top-concept-of": {
+          "predicate": "skos:topConceptOf",
+          "target": "concepts",
+          "cardinality": "one"
+        },
+        "has-broader-role": {
+          "predicate": "skos:broader",
+          "target": "membership-roles",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/id/rollen/"
+    },
```

Comment:
**Blocker:** `org:Role` is already defined https://github.com/lblod/app-organization-portal/blob/5ff3f5b1219f5c58c8cc686c76c32c3484da097b/config/resources/domain.json#L358-L366

Reply:
Use the skos concept scheme?